lib.go

```diff
@@ -525,6 +531,15 @@ func (vm *VM) IBCPacketTimeout(
 	return &result, gasReport.UsedInternally, nil
 }
 
+func compileCosts(code WasmCode) uint64 {
```

Comment:
This should probably be singular

```suggestion
func compileCost(code WasmCode) uint64 {
```